includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -29,7 +29,9 @@ class DesignSystemGlobalFooterModel extends WikiaModel {
 			'social-twitter' => 'https://twitter.com/wikia',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "social-twitter" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)